python/GafferTest/TweakPlugTest.py

```diff
@@ -354,6 +354,44 @@ def testTweakInternedString( self ) :
 		self.assertTrue( result )
 		self.assertEqual( data["a"], IECore.InternedStringData( "stringValue" ) )
 
+	def testPathMatcherListOperations( self ) :
+
+		data = IECore.CompoundData(
+			{
+				"p" : IECore.PathMatcherData()
+			}
+		)
+
+		createTweak = Gaffer.TweakPlug( "p", IECore.PathMatcherData( IECore.PathMatcher( [ "/test/path", "/test/path2" ] ) ) )
```

Comment:
Oh, but one UI tweak we should probably make now is to update https://github.com/GafferHQ/gaffer/blob/main/python/GafferUI/TweakPlugValueWidget.py#L185-L201 so that it includes the List modes for PathMatcherDataPlugs.

Reply:
Tweaked in https://github.com/GafferHQ/gaffer/pull/4907/commits/e7ebd724c89c4cf7a37e6d36de7fab25fd384d3d